### PR TITLE
modified readme and SEO improvements

### PR DESCRIPTION
**Pull request by ananyo2012**
Created first pull request.Modified readme.

### CODE REVIEW

**Comment by sonalkr132** [comment]
Thanks @ananyo2012 

Can you please update this so that readme has information about all the example file? Besides devise, I think there is cloudinary.rb and production.rb

**Comment by ananyo2012** [comment]
I have added new commits to my previous pull request.changed  the title and meta data and also modified the readme as you said.

File: app/views/users/passwords/edit.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<h2>Change your password</h2>
+<% @pageTitle = "Set new Password | CAT Forum" %>
+<% @PageDescription = "Set new password for your account" %><h2>Change your password</h2>
```
Comment (sonalkr132) [inline app/views/users/passwords/edit.html.erb:4]: why did you remove content of h2 tag here?

**Comment by ananyo2012** [comment]
no i didn't remove it actually.it is there but not indented properly.the changes are in the same line actually.i didn't observe it before properly so i didn't give the indentation well.

**Comment by sonalkr132** [comment]
Can you please point me to the line where it says 'change your password' under h2 tag? As far as I can see it is not there https://github.com/ananyo2012/cat-forum/blob/github_master/app/views/users/passwords/edit.html.erb

File: app/views/users/passwords/edit.html.erb (hunk 1 above)
Comment (ananyo2012) [inline app/views/users/passwords/edit.html.erb:4]: please see beside  `<% @PageDescription = "Set new password for your account" %>` in line 2.the h2 tag is after the erb tag.

Comment (sonalkr132) [inline app/views/users/passwords/edit.html.erb:4]: Please fix the indentation. Now that many people are working on same
project it is important that we try to keep our code clean.

**Comment by ananyo2012** [comment]
added microdata and modified the robots.txt to disallow the admin area of cat-forum.also fixed the previous indentation and changed the name of the image cat.jpg to CAT-Forum .jpg and made the necessary changes.

**Comment by sonalkr132** [comment]
@ghoshbishakh @ananyo2012 Just so that everyone knows. We will be using 2 spaces (not tab) for indentation. Configure your editor accordingly.

**Comment by ananyo2012** [comment]
ok will be taking care for next pull requests
